isis/CMakeLists.txt

```diff
@@ -452,11 +452,22 @@ execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/inc)
 # Create the appdata directory and translations directory
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/translations)
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/apollo)
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/autoreg) # *.def
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/autoseed) # *.def, *.pvl
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/cnetref) # *.def, *.def.example
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/cnetstats) # *.def, *.def.example
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/cnet_validmeasure) # *.def, *.def.example
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/findfeatures) # *.lis
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/fullcnetdiff) # *.lis
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/jigsaw) # *.pvl
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/kernels) # *.txt
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/photometry) # *.pvl
+
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/maps)
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/images/icons)
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/images/targets)
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/labels)
-execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/autoreg)
+#execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/autoreg)
```

Comment:
I moved autoreg from $ISISROOT/appdata/autoreg to $ISISROOT/appdata/templates/autoreg because it made more sense to me. Let me know if you agree or disagree.

Reply:
It does